test/test-suite/common/error/SensorAccessLinkErrorTest.cpp

```diff
@@ -45,7 +45,6 @@ class SensorAccessLinkErrorTest : public ::testing::Test {
 };
 
 // TODO : If error = ALARM, do no reopen
```

Comment:
Tu cherches a faire quoi avec ce TODO est il toujours utile ?

Reply:
It's done now, un oubli de ma part, good catch @RphlBrdt